lib/Service/RowService.php

```diff
@@ -223,7 +223,7 @@ private function parseValueByColumnType(Column $column, $value = null) {
 		} catch (NotFoundExceptionInterface|ContainerExceptionInterface $e) {
 			$this->logger->debug('Column type business class not found', ['exception' => $e]);
 		}
-		return "".$value;
+		return (string) $value;
```

Comment:
Same can be done in line 220. 

In line 222 we can legitimately get mixed return types, right?